packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -206,6 +206,30 @@ const releaseApi = createApi({
         invalidatesTags: (result, error, arg) => [
           { type: 'ReleaseAction', id: arg.params.actionId },
         ],
+        async onQueryStarted({ body, params, query }, { dispatch, queryFulfilled }) {
+          // We need to mimic the same params received by the getReleaseActions query
+          const paramsWithoutActionId = {
+            releaseId: params.releaseId,
+            ...query,
+          };
+
+          const patchResult = dispatch(
+            releaseApi.util.updateQueryData('getReleaseActions', paramsWithoutActionId, (draft) => {
+              // @ts-expect-error Type instantiation is excessively deep and possibly infinite.
+              const action = draft.data.find((action) => action.id === params.actionId);
```

Comment:
I have 0 idea about this TS error, so any clue would be appreciate 😅

Reply:
It's because you're looking for a ReleaseAction, and a ReleaseAction can have `release: Release;` and a Release has `actions: ReleaseAction[];` and you can see where i'm going with this.